netsim/devices/iol.yml

```diff
@@ -0,0 +1,100 @@
+---
+description: Cisco IOL
+interface_name: Ethernet{ifindex // 4}/{ifindex % 4}
```

Comment:
Is this correct? The only other platform using EthernetX/Y naming scheme is IOSv

Reply:
show ip int br

Ethernet0/0            192.168.121.101 YES TFTP   up                    up      
Ethernet0/1            10.1.0.1        YES manual up                    up      
Ethernet0/2            10.1.0.5        YES manual up                    up      
Ethernet0/3            10.1.0.9        YES manual up                    up      
Ethernet1/0            10.1.0.13       YES manual up                    up      
Ethernet1/1            10.1.0.17       YES manual up                    up      
Ethernet1/2            10.1.0.21       YES manual up                    up      
Ethernet1/3            10.1.0.25       YES manual up                    up      
Ethernet2/0            10.1.0.29       YES manual up                    up      
Ethernet2/1            10.1.0.33       YES manual up                    up